### ISSUE TITLE
Fine-tune the model on BridgeV2

### ISSUE THREAD

**Issue by hutchinsonian** [openvla/openvla#98]
I tried to fine-tune the model using the following script, but why is the loss increasing? I saw in the repo that the author mentioned that because the pretrain training data includes BridgeV2, the accuracy should be close to 100% during fine-tuning. I observed this phenomenon in the first few steps, but the accuracy kept decreasing afterwards. 
Why is this? Is this normal? Am I overlooking 
```
CUDA_VISIBLE_DEVICES=6 torchrun --standalone --nnodes 1 --nproc-per-node 1 vla-scripts/finetune.py \
  --vla_path "hugg_models/openvla-7b" \
  --data_root_dir "datasets" \
  --dataset_name bridge_orig \
  --run_root_dir "checkpoints_log" \
  --adapter_tmp_dir "checkpoints_log/adapter_tmp" \
  --lora_rank 32 \
  --batch_size 4 \
  --grad_accumulation_steps 4 \
  --learning_rate 5e-4 \
  --image_aug False \
  --wandb_project xx \
  --wandb_entity xx \
  --save_steps 2500
```
<img width="1079" alt="image" src="https://github.com/user-attachments/assets/871662d6-33b0-4f85-9c0e-6b497a320365">

**Comment by kpertsch** [comment]
Mmh this is very odd, esp since we're logging *training* metrics, so it's surprising that the loss is going *up*.
My best guess: your batch size is too low and your learning rate is too high. Maybe you can try a run where you use a larger effective batch size (eg 128) and a lower learning rate (eg 1e-5) and see whether that works better.
